frontend/src/components/SearchHitHeader.vue

```diff
@@ -1,28 +1,37 @@
 <template>
    <div class="header-wrapper">
-      <div class="full-title">
-         <span class="count-wrap" v-if="count">
-            <span class="count">{{count}}</span><span v-if="subcount" class="count sub">.{{subcount}}</span>
-         </span>
-         <template v-if="link == false">
-            <span class="hit-title" v-html="hit.header.title"></span>
-            <span v-if="hit.header.subtitle" class="hit-subtitle" v-html="hit.header.subtitle"></span>
-         </template>
-         <router-link @click.native="detailClicked" v-else :to="detailsURL">
-            <span class="hit-title" v-html="hit.header.title"></span>
-            <span v-if="hit.header.subtitle" class="hit-subtitle" v-html="hit.header.subtitle"></span>
-         </router-link>
+      <div class="title-wrapper">
+         <div class="full-title">
+            <span class="count-wrap" v-if="count">
+               <span class="count">{{count}}</span><span v-if="subcount" class="count sub">.{{subcount}}</span>
+            </span>
+            <template v-if="link == false">
+               <span class="hit-title" v-html="hit.header.title"></span>
+               <span v-if="hit.header.subtitle" class="hit-subtitle" v-html="hit.header.subtitle"></span>
+            </template>
+            <router-link @click.native="detailClicked" v-else :to="detailsURL">
+               <span class="hit-title" v-html="hit.header.title"></span>
+               <span v-if="hit.header.subtitle" class="hit-subtitle" v-html="hit.header.subtitle"></span>
+            </router-link>
+         </div>
+         <div class="bm-control">
+            <AddBookmark v-if="isSignedIn" :hit="hit" :pool="pool" :id="`bm-modal-${hit.identifier}`"/>
+            <SignInRequired v-else  :hit="hit" :id="`bm-modal-${hit.identifier}`" act="bookmark" />
+         </div>
       </div>
-      <div class="bm-control">
-         <AddBookmark v-if="isSignedIn" :hit="hit" :pool="pool" :id="`bm-modal-${hit.identifier}`"/>
-         <SignInRequired v-else  :hit="hit" :id="`bm-modal-${hit.identifier}`" act="bookmark" />
+      <div v-if="hit.header.author_display" class="author-wrapper">
```

Comment:
not sure I understand what is goin on here?

Reply:
I wanted to add a div outside of the title (so that the author would be on a separate line), but Vue requires just one div under the template.  So I essentially renamed header-wrapper to title-wrapper, then added a new header-wrapper that included title-wrapper and author-wrapper.  Then I played with padding values until the author looked pretty good on both search and item details pages.

Reply:
more unsure of the if for author under the if for author_display. looks like author_display set, but not author loses any author info

Reply:
author_display is set to the joined author string in preProcessHitFields() if not explicitly provided by the pool, so it's guaranteed to exist if the pool provided any author or author-display type values.  I guess the v-if is there in case the pool provided neither, which is theoretically possible.

Reply:
oops... I missed the ! in the last change in preProcessHits... makes sense